src/components/molecules/Visualizer/Engine/ref.ts

```diff
@@ -61,6 +62,7 @@ export type EngineRef = {
   zoomIn: (amount: number) => void;
   zoomOut: (amount: number) => void;
   changeSceneMode: (sceneMode: SceneMode | undefined, duration?: number) => void;
+  getClock: () => Clock | undefined;
```

Comment:
It has become dependent on Cesium, so it should be a generic type independent from the specific map engine.

e.g.

```ts
type Clock = {
  isPlaying: boolean;
  setPlaying: (playing: boolean) => void;
  currentTime: Date; // note that it should be JS Date
};
```

Reply:
I fixed this comment.
[fix: abstract clock value](https://github.com/reearth/reearth-web/pull/285/commits/8ef72f3e3075b397e1130d9042b037333cbc6690)